public/playpass-content.schema.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
this schema is missing fields that are detailed in the documentation and are in use in the template

https://github.com/playpassgames/daily-audio-game-template#configuration-contentsongsjson

Reply:
Ah that changes things quite a bit
I'll update the schema to reflect this and we can sync later on what we want to be configurable for the MVP